internal/command/idp.go

```diff
@@ -52,3 +58,34 @@ type LDAPProvider struct {
 	LDAPAttributes      idp.LDAPAttributes
 	IDPOptions          idp.Options
 }
+
+func ExistsIDP(ctx context.Context, filter preparation.FilterToQueryReducer, id, orgID string) (exists bool, err error) {
```

Comment:
Should this be a copy of the logic used for org and instance? As you can return anyway if found you could use the other 2 functions for less doublicate code.